x-pack/test/security_solution_cypress/serverless_config.ts

```diff
@@ -29,6 +29,10 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ...svlSharedConfig.get('kbnTestServer.serverArgs'),
         '--serverless=security',
         '--xpack.encryptedSavedObjects.encryptionKey="abcdefghijklmnopqrstuvwxyz123456"',
+        `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+          'riskScoringRoutesEnabled',
```

Comment:
I'll chime in here since @MadameSheema didn't: I'm guessing you added these during local development due to https://github.com/elastic/security-team/issues/7423.

It's sorta a moot point since we're going to change the default values here shortly, but let's try to remember to remove these once that happens.